src/pages/PieIndexLanding.svelte

```diff
@@ -137,6 +136,8 @@
     }),
   );
 
+  $: console.debug({ composition })
```

Comment:
is this something should go on production?

Reply:
no, thank you good spot